packages/forms-web-app/src/dynamic-forms/dynamic-components/list-add-more/question.js

```diff
@@ -321,6 +321,32 @@ class ListAddMoreQuestion extends Question {
 	async removeAction(req, journeyResponse, answerId) {
 		return this.subQuestion.removeList(req, journeyResponse, answerId);
 	}
+
+	/**
+	 * @param {JourneyResponse} journeyResponse
+	 * @returns {boolean}
+	 */
+	isAnswered(journeyResponse) {
```

Comment:
You're probably wondering why the heck I did this here. Honestly it didn't need doing immediately but by creating an isAnswered method on questions I was able to cut out a lot of logic from `Section.getStatus` (which is where this code is ripped from) and remove Sections dependencies on add more questions and validators.